### ISSUE TITLE
failed push for alt text spread sheet

### ISSUE THREAD

**Issue by kimberly-bastille** [andybeet/ESR#3]
I was unable to push the csv version of the alt text spreadsheet. Let me know if you need it in another format.

[ESR_alt_text.xlsx](https://github.com/andybeet/ESR/files/4746538/ESR_alt_text.xlsx)

**Comment by andybeet** [comment]
you are now a collaborator.
